#ifndef WINDOWMAIN_H
#define WINDOWMAIN_H

#include <QMainWindow>

QT_BEGIN_NAMESPACE
namespace Ui { class WindowMain; }
QT_END_NAMESPACE

class WindowMain : public QMainWindow
{
    Q_OBJECT
public:
    WindowMain(QWidget *parent = nullptr);
    ~WindowMain();

private:
    Ui::WindowMain* v_ui = nullptr;

};
#endif // WINDOWMAIN_H
